app/build.gradle.kts

```diff
@@ -131,6 +131,8 @@ dependencies {
     androidTestImplementation(libs.kotlin.test)
 
     baselineProfile(projects.benchmarks)
+
+    lintChecks(libs.android.security.lint)
```

Comment:
I don't think lint is currently configured to check dependencies, and merge the reports. This would mean that this `lintCheck` will only be used for the app module.

Reply:
Thanks @SimonMarquis, I think it's configured here?https://github.com/android/nowinandroid/blob/d15c739812f25401b21614fe0f7e18534d285921/build-logic/convention/src/main/kotlin/AndroidLintConventionPlugin.kt#L46

Reply:
Oh you're right, I completely forgot we had it in here. Then I guess this is good 👍